docs/product/roadmap.md

```diff
@@ -29,33 +29,43 @@ Additional value here is preventing entities that don't meet the criteria from s
 
 **Specific deliverables:**
 * (DONE ✅) Technical foundation
+* (DONE ✅) Login.gov integration
 * (90% ✅) Create a new audit (pre SAC forms)
-* (90% ✅) Login.gov integration
 * (60% ✅) design/exploration - "Submit an entire audit package" milestone
 
 ## Milestone: Submit general information end to end / De-risk
 
 **Value delivered:**
 To de-risk our product plans, we will focus on an end-to-end solution that bridges data input to dissemination to the current FAC. This will include a fully fleshed out SF-SAC general information page. We will also build out the API to de-risk unknows in end-to-end implementation and unblock our partners at Census by providing specifics of API format. 
 
+Submit general information end to end:
 * (20% ✅) [General information - SF-SAC data entry](https://github.com/GSA-TTS/FAC/issues/185)
-* [API to transfer data to Census](https://github.com/GSA-TTS/FAC/issues/310)
-* [PDF Validation Spike](https://github.com/GSA-TTS/FAC/issues/307)
-* (We've already done a spike around Excel file ingestion)
-* (60% ✅) design/exploration - "Submit an entire audit package" milestone
```

Comment:
This is very close to done, but is dependent on reopening the Q of whether we'll be accepting data vis web form as well as Excel in the initial release.

Reply:
Updated